src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -403,6 +403,11 @@ case class IndexLogEntry(
     sourcePlanSignatures.head
   }
 
+  def excludedFiles: Seq[String] = {
+    // Only one relation is currently supported (See relations).
```

Comment:
We can remove this comment. The `assert` has been moved to `relations` to remove duplicate comments like this in the previous PR - I forgot which one).